### PR TITLE
fix ttl test failed in windows, split test functions

### PR DESCRIPTION
**Pull request by a-breid-ericsson**
Fixed Test TTL for Automatic Subscription Trigger to work in  windows.

Since it was hard to follow one of the test steps as it did three different things I split one step to 3 steps.

Run test using command: `mvn -Dtest=TestTTLRunner test -Dcucumber.options="--tags @TestTTLforAutomaticSubscriptionTrigger"`

Changed all taps to 4 spaces.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by:

### CODE REVIEW

File: src/functionaltests/java/com/ericsson/ei/notifications/ttl/TestTTLSteps.java [hunk 1]
```diff
@@ -42,196 +42,197 @@
...
 @ContextConfiguration(initializers = TestContextInitializer.class)
 public class TestTTLSteps extends FunctionalTestBase {
-	private static final Logger LOGGER = LoggerFactory.getLogger(TestTTLSteps.class);
-	private static final String BASE_URL = "localhost";
-	private static final String ENDPOINT = "/missed_notification";
-	private static final String SUBSCRIPTION_NAME = "Subscription_1";
+    private static final Logger LOGGER = LoggerFactory.getLogger(TestTTLSteps.class);
+    private static final String BASE_URL = "localhost";
+    private static final String ENDPOINT = "/missed_notification";
+    private static final String SUBSCRIPTION_NAME = "Subscription_1";
```
Comment (tobiasake) [inline src/functionaltests/java/com/ericsson/ei/notifications/ttl/TestTTLSteps.java]: Which code parts has changed? everything is green marked.

Comment (Christoffer-Cortes) [reply src/functionaltests/java/com/ericsson/ei/notifications/ttl/TestTTLSteps.java]: You can remove tab changes by clicking Diff settings-> Hide whitespace changes

File: src/functionaltests/java/com/ericsson/ei/notifications/ttl/TestTTLSteps.java [hunk 2]
```diff
@@ -42,196 +42,197 @@
...
+    }
+
+    @When("^Missed notification is created$")
+    public void a_missed_notification_is_created() throws Throwable {
+        // verifying that missed notification is created and present in db
+        String condition = "{\"subscriptionName\" : \"" + SUBSCRIPTION_NAME_3 + "\"}";
+        long maxTime = System.currentTimeMillis() + MAX_WAIT_TIME;
+        List<String> notificationExit = null;
+        LOGGER.debug("Checking presence of missnotification in db");
+        while (System.currentTimeMillis() < maxTime) {
```
Comment (evasiba-ericsson) [inline src/functionaltests/java/com/ericsson/ei/notifications/ttl/TestTTLSteps.java]: Please extract the piece of code that fetches existing notifications and create a method of it. Then reuse it in this and in the following step. 

Fix variable naming from notificationExit to notificationExist.

Comment (a-breid-ericsson) [reply src/functionaltests/java/com/ericsson/ei/notifications/ttl/TestTTLSteps.java]: Although this is out of the scope (if you disable white space changes in diff settings) I may look in to this.
